app/models/wizards/signup/sektion_operation.rb

```diff
@@ -24,6 +24,8 @@ def save!
       save_person_and_role
       generate_invoice if no_approval_needed? && can_receive_invoice?
       exclude_from_mailing_list if mailing_list && !newsletter
+      enqueue_duplicate_locator_job
+      enqueue_notification_email
```

Comment:
Hier fehlt noch  das enqueuen vom password reset email

Reply:
das habe ich hinzugefügt, aber nachher wieder entfernt, weil das password reset email schon ohne versendet wird. sonst würde es 2x senden

Reply:
kannst du das testen auf localhost?

Reply:
Kann es sein, dass zu dem Zeitpunkt der controller auch noch das Email verschickt hat? Sonst kann ich es mir nicht erklären. Die tests und lokales ausprobieren meint auch, für jede email in der operation pw reset verschicken ist okay.